src/docs/SUPPORTED_MODELS.md

```diff
@@ -193,6 +193,14 @@ The pipeline can work with other similar topologies produced by `optimum-intel`
         </ul>
       </td>
     </tr>
+    <tr>
+      <td><code>Stable Diffusion 3</code></td>
+      <td>
+        <ul>
+          <li><a href="https://huggingface.co/stabilityai/stable-diffusion-3-medium-diffusers"><code>stabilityai/stable-diffusion-3-medium-diffusers</code></a></li>
```

Comment:
let's also check `stabilityai/stable-diffusion-3.5-large` and `stabilityai/stable-diffusion-3.5-large-turbo` (we can do it separately)

they should be supported as well

Reply:
Let's create a separate ticket to support these models

Reply:
CVS-156249